.github/workflows/on-push.yml

```diff
@@ -5,14 +5,23 @@ env:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        otp: ['23.3.4.11']
+        elixir: ['1.9.4', '1.13.2']
     steps:
       - uses: actions/checkout@v2
       - uses: erlef/setup-elixir@v1
         with:
-          otp-version: '23.0.4'
-          elixir-version: '1.10.4'
+          otp-version: ${{matrix.otp}}
+          elixir-version: ${{matrix.elixir}}
       - uses: rrainn/dynamodb-action@v2.0.0
+      - run: mix deps.update --all
```

Comment:
I don't think we'd want to call `mix deps.update` in the Github workflow, because that unlocks and updates the dependency versions specified in mix.lock, and so I think you could end up running the workflow against different versions of the dependencies than are specified in the checked-in mix.lock.

I would think that we just want to call `mix deps.get` so we install whatever dependency versions are in the checked-in mix.lock - so you could remove lines 19 and 20, if that's correct.

Reply:
OK. I'll create other PR.